test/cli_test_runner.cc

```diff
@@ -0,0 +1,518 @@
+#include "doctest.h"
```

Comment:
The name of this file is confusing. It doesn't run `cli` tests.
test_corpus_runner?

Reply:
Sure, happy to change it.

Reply:
Actually, how about `pipeline_test_runner.cc`? This file doesn't run the LSP or Parser tests anymore, so I'd like it to have a more specific name.

Reply:
Sure!